src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder sb = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```

Comment:
String builder - is not a service class. It is just an improved version of the string.
```suggestion
        StringBuilder result = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```
Also, it is a bad practice to use abbreviations.